### PR TITLE
React gui

### PR DESCRIPTION
**Pull request by zalash333**

### CODE REVIEW

File: src/iob [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+require('/Users/zalash/MyProject/ioBroker.web/src/node_modules/iobroker.js-controller/lib/setup').execute();
```
Comment (Apollon77) [inline src/iob:2]: is that really wanted in the PR?

File: src/.eslintcache [hunk 2]
```diff
@@ -0,0 +1 @@
...Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Tabs/WhiteList.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Tabs/Options.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Tabs/Certificates.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Tabs/Background.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Components/CustomSelect.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Components/CustomButtonUpload.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Components/CustomCheckbox.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Components/CustomInput.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Components/HintComponent.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Components/CustomModal.js",[],"/Users/zalash/MyProject/Новая папка/ioBroker.web/src/src/Components/Toast.js",[],{"ruleId":"117","replacedBy":"118"},{"ruleId":"119","replacedBy":"120"},{"ruleId":"117","replacedBy":"121"},{"ruleId":"119","replacedBy":"122"},"no-native-reassign",["123"],"no-negated-in-lhs",["124"],["123"],["124"],"no-global-assign","no-unsafe-negation"]
```
Comment (Apollon77) [inline src/.eslintcache:1]: wanted in PR?

File: src/iobroker [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
```
Comment (Apollon77) [inline src/iobroker:1]: same

File: src/public/blockly.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
+'use strict';
```
Comment (Apollon77) [inline src/public/blockly.js:1]: really wanted here?

File: src/reinstall.js [hunk 5]
```diff
@@ -0,0 +1,158 @@
+'use strict';
```
Comment (Apollon77) [inline src/reinstall.js:1]: really wanted?

**Comment by Apollon77** [comment]
@zalash333 Can you please tell some more about the purpose of this PR?

**Review by AlCalzone — CHANGES_REQUESTED** [review]
I think the dotfiles should be gitignored and not be in here.

Can't speak for the rest - I don't have the capacity right now to review this in depth.

**Pull request merged**